hedera-node/hedera-mono-service/src/main/java/com/hedera/services/grpc/marshalling/AliasResolver.java

```diff
@@ -70,6 +73,16 @@ public CryptoTransferTransactionBody resolve(
         return resolvedOp.build();
     }
 
+    public void resolve(final EthTxData ethTxDataMeta, final AliasManager aliasManager) {
```

Comment:
Since we don't actually resolve the `EthTxData` to a different type or transaction body, probably we can rename the method to something like `perceiveEthTransaction()`?

Reply:
Agreed, renamed now